server/src/main/java/org/apache/gravitino/server/web/rest/ModelOperations.java

```diff
@@ -101,10 +101,10 @@ public Response listModels(
             modelIds =
                 MetadataFilterHelper.filterByExpression(
                     metalake,
-                    "ANY(OWNER,METALAKE,CATALOG,SCHEMA) || "
+                    "ANY(OWNER,METALAKE,CATALOG) || "
                         + "((ANY(USE_CATALOG,METALAKE, CATALOG)) "
-                        + "&& (ANY(USE_SCHEMA,METALAKE, CATALOG, SCHEMA)) "
-                        + "&& (MODEL::OWNER||ANY(USE_MODEL,METALAKE, CATALOG, SCHEMA,MODEL)))",
+                        + "&& (SCHEMA::OWNER || (ANY(USE_SCHEMA,METALAKE, CATALOG, SCHEMA)) "
```

Comment:
`USE_CATALOG` instead `USE_SCHEMA`. The owner of the schema have already had `USE_SCHEMA`.

Reply:
Here indicates that the Model Owner needs to USE_SCHEMA, and both the MODEL OWNER and SCHEMA OWNER need to USE_CATALOG.